recipe/meta.yaml

```diff
@@ -4,9 +4,10 @@ about:
 
 package:
   name: multiply-sar-preprocessing
-  version: "0.1"
```

Comment:
We should use version 0.1 here, too. Otherwise, set it to 0.4 in version.py and changes.md, but we need the same version everywhere.

Reply:
Agreed. Changed to 0.1